samples/contrib/azure-samples/databricks-pipelines/databricks_operator_cluster_pipeline.py

```diff
@@ -0,0 +1,108 @@
+"""Create a cluster in Databricks. Then submit a one-time Run to that cluster."""
+import kfp.dsl as dsl
+import kfp.compiler as compiler
+
+def create_cluster(cluster_name):
+    return dsl.ResourceOp(
+        name="createcluster",
+        k8s_resource={
+            "apiVersion": "databricks.microsoft.com/v1alpha1",
+            "kind": "Dcluster",
+            "metadata": {
+                "name":cluster_name,
+            },
+            "spec":{
+                "cluster_name": cluster_name,
+                "spark_version": "5.3.x-scala2.11",
+                "node_type_id": "Standard_D3_v2",
+                "spark_conf": {
+                    "spark.speculation": "true"
+                },
+                "num_workers": 2
+            }
+        },
+        action="create",
+        success_condition="status.cluster_info.state in (RUNNING, TERMINATED, UNKNOWN)",
+        attribute_outputs={
+            "name": "{.metadata.name}",
+            "cluster_id": "{.status.cluster_info.cluster_id}",
+            "cluster_name": "{.status.cluster_info.cluster_name}",
+            "state": "{.status.cluster_info.state}"
+        }
+    )
+
+def submit_run(run_name, cluster_id, parameter):
+    return dsl.ResourceOp(
+        name="submitrun",
+        k8s_resource={
+            "apiVersion": "databricks.microsoft.com/v1alpha1",
+            "kind": "Run",
+            "metadata": {
+                "name": run_name,
+            },
+            "spec":{
+                "run_name": run_name,
+                "existing_cluster_id": cluster_id,
+                "libraries": [{"jar": "dbfs:/my-jar.jar"}],
```

Comment:
Just curious, does this jar compute pi? In another file it is `dbfs:/docs/sparkpi.jar`

Reply:
I just removed all databricks_operator_* samples, as the idea is for people to use our Azure Databricks for Kubeflow Pipelines package.